### PR TITLE
Add test for empty io in BLS

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: qa/python_models/bls/model.py [hunk 1]
```diff
@@ -140,6 +140,20 @@ def _get_gpu_bls_outputs(self, input0_pb, input1_pb):
 
         return output0.to_dlpack(), output1.to_dlpack()
 
+    def test_zero_length_io(self):
+        model_name = 'identity_fp32'
+        input0 = np.zeros([1, 0], dtype=np.float32)
```
Comment (CoderHam) [inline qa/python_models/bls/model.py:145]: Is not `input0 = np.zeros([0], dtype=np.float32)` sufficient?

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: qa/python_models/bls/model.py (hunk 1 above)
Comment (Tabrizian) [reply qa/python_models/bls/model.py:145]: @CoderHam Since identity model has `max_batch_size > 0`  it needs to have two dims.

**Review by CoderHam — APPROVED** [review]
